src/db/mongo.rs

```diff
@@ -179,4 +185,28 @@ impl MeigenDatabase for MongoMeigenDatabase {
             .context("returned document's id property wasn't i64")
             .map(|x| x as u32)
     }
+
+    async fn append_loved_user(&mut self, id: u32, loved_user_id: u64) -> Result<bool> {
+        self.inner
+            .update_one(
+                doc! { "id": id },
+                doc! { "$addToSet": { "loved_user_id": loved_user_id as u32 } },
+                None
+            )
```

Comment:
casting `loved_user_id` wouldn't work here because usually user id is bigger than u32::MAX and information loss occurs.
so I think storing user id as String is fair. Convert to String before serialize, and call `parse()` after deserialize. Please use `MongoMeigen` to do that. I think changing `From` to `TryFrom` works well.

```suggestion
                doc! { "id": id },
                doc! { "$addToSet": { "loved_user_id": loved_user_id.to_string() } },
                None
            )
```

Reply:
[f61eea1](https://github.com/approvers/MeigenBot-Rust/pull/17/commits/f61eea18bd1f99c99c2630126ea82f7970db0776)